src/appleseed/foundation/math/microfacet.cpp

```diff
@@ -224,12 +224,12 @@ float BlinnMDF::pdf(
 //
 
 float BeckmannMDF::D(
-    const Vector3f&     h,
+    const Vector3f&     m,
```

Comment:
Here too

Reply:
The first part of this file is code that's in an unnamed namespace.
It doesn't need to have the same parameters alignment as the rest of the code in the file.

Reply:
Ok :)